requirements/collections_requirements.yml

```diff
@@ -1,22 +1,12 @@
 ---
 collections:
   - name: awx.awx
-    version: 11.2.0  # almost all versions should work
   - name: azure.azcollection
-    version: 0.1.2
   - name: amazon.aws
-    version: 0.1.1  # version 0.1.0 seems to have gone missing
   - name: theforeman.foreman
-    version: 0.8.1
   - name: google.cloud
-    version: 0.0.9  # contains PR 167, should be good to go
   - name: openstack.cloud
-    version: 0.0.1-dev85  # earlier had checksum mismatch
```

Comment:
question I would have asked is if this works with the `-dev` versions, but since this was created it looks like openstack.cloud has released a non-dev version number, so that should no longer be a stumbling block.

Reply:
oo, good catch, and thanks for checking on the collection 👍